s4e-web/src/app/views/settings/groups/state/group.service.ts

```diff
@@ -23,7 +23,13 @@ export class GroupService {
     this.store.setLoading(true);
     this.store.setError(null);
     this.http.get<IPageableResponse<Institution>>(`${this.config.apiPrefixV1}/institutions/${institutionSlug}/groups`)
-      .pipe(finalize(() => this.store.setLoading(false)))
+      .pipe(
+        finalize(() => this.store.setLoading(false)),
+        catchError(error => {
+          this.store.setError(error);
+          return throwError(error);
+        })
```

Comment:
This happens very often now, I think that you can consider making our own RXJS operator (e.g. `catchStoreAndThrow(store)`)

Reply:
I think it's better to consider such changes during the refactoring session. For now, the whole feature is big enough to be considered as the only thing to be done.